slack-api-client/src/main/java/com/slack/api/methods/SlackApiErrorResponse.java

```diff
@@ -10,4 +13,5 @@
     private String error;
     private String needed;
     private String provided;
+    private transient Map<String, List<String>> httpResponseHeaders;
```

Comment:
Having transient tells the JSON library that this property should not be included in serialized JSON string data. No side effect for other use cases.